controllers/front/CartController.php

```diff
@@ -637,7 +637,7 @@ protected function areProductsAvailable()
         if ($product['active']) {
             return $this->trans(
                 'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.',
-                ['%product%' => $product['name']],
+                ['%product%' => $product['name'] . (isset($product['attributes']) ? ' ' . $product['attributes'] : '')],
```

Comment:
Hi @JBWModules I'm wondering, with this works with multilang products? Should'nt we pick the right localized strings, something that look like `$product['attributes'][$locale]` ?

Reply:
It's already translated in the active lang (same as $product['name'] was before).

Reply:
Thanks for the answer, code seems good for me 😊